spec/paths/reports@subscription-cancellation.yaml

```diff
@@ -0,0 +1,54 @@
+get:
+  tags:
+    - Reports
+  summary: Retrieve transactions with their plan data
+  description: Retrieve transactions with their plan data
+  parameters:
+    - name: periodStart
+      in: query
+      description: Report start date
+      type: string
+      required: true
+    - name: periodEnd
+      in: query
+      description: Report end date
+      type: string
+      required: true
+    - name: aggregationField
+      in: query
+      description: Report aggregation field
+      required: true
+      type: string
+      enum:
+        - canceled-by
+        - cancel-category
+        - leads.affiliate
```

Comment:
@XaosSintez where I can see such examples? Every single resource I see have it in camel case like `/customers/?filter=firstName:John`.